### ISSUE TITLE
GitHub TestSuite checks are failing

### ISSUE THREAD

**Issue by adnaanbheda** [Kludex/uvicorn#1713]
I think this might be due to #1696, the pip upgrade fails for Windows, and others are cascaded into cancellation.

```
Run scripts/install
  scripts/install
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\3.7.9\x64
    PKG_CONFIG_PATH: C:\hostedtoolcache\windows\Python\3.7.9\x64/lib/pkgconfig
    Python_ROOT_DIR: C:\hostedtoolcache\windows\Python\3.7.9\x64
    Python[2](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:2)_ROOT_DIR: C:\hostedtoolcache\windows\Python\[3](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:3).7.9\x6[4](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:4)
    Python3_ROOT_DIR: C:\hostedtoolcache\windows\Python\3.7.9\x[6](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:6)4
+ '[' -z true ']'
+ PIP=pip
+ pip install -U pip
Requirement already satisfied: pip in c:\hostedtoolcache\windows\python\3.[7](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:7).[9](https://github.com/encode/uvicorn/actions/runs/3264291960/jobs/5364701055#step:4:9)\x64\lib\site-packages (22.2.2)
Collecting pip
  Downloading pip-22.3-py3-none-any.whl (2.1 MB)
ERROR: To modify pip, please run the following command:
     ---------------------------------------- 2.1/2.1 MB 32.4 MB/s eta 0:00:00
c:\hostedtoolcache\windows\python\3.7.9\x64\python.exe -m pip install -U pip

Notice:  A new release of pip available: 22.2.2 -> 22.3
Notice:  To update, run: python.exe -m pip install --upgrade pip
Error: Process completed with exit code 1.
```

**Comment by Kludex** [comment]
Amazing.

**Comment by adnaanbheda** [comment]
The test didn't fail at that time because `22.2.2` was the latest version then lol (I think)

**Comment by Kludex** [comment]
- Closed by #1715
